keytransparency/BUILD

```diff
@@ -0,0 +1,28 @@
+android_library(
+  name = "keytransparency",
+  visibility = ["//visibility:public"],
+  srcs = glob(["src/main/java/com/google/keytransparency/*.java"]),
+  custom_package = "com.google.keytransparency",
+  manifest = "src/main/AndroidManifest.xml",
+  resource_files = glob(["src/main/res/**"]),
+  deps = [":gobind_keytransparency_android"],
+)
+
+aar_import(
+        name = "gobind_keytransparency_android",
+        aar = "keytransparency_gobind.aar",
+    )
+
+genrule(
+    name = "gobind_keytransparency_android_aar",
+    srcs = [
+       # "//some:files",  # a filegroup with multiple files in it ==> $(locations)
+       # "//other:gen",   # a genrule with a single output ==> $(location)
+    ],
+    outs = ["keytransparency_gobind.aar"],
+    tags=["local"],
+    local = 1,
+    cmd = "gomobile bind -target android -o $@ -javapkg com.google.keytransparency.gobind github.com/google/keytransparency/core/client/gobindClient github.com/google/keytransparency/core/client/kt github.com/google/keytransparency/core/crypto/commitments",
```

Comment:
I think so (so that the library is rebuilt when they change), but they live in the $GOPATH directory which is not part of the workspace. 
Point is that these packages in turn depend on other go packages (like trillian), so I would have to list the whole GOPATH as source? 

Also, we need to decide what to do with vendoring....